kmelia/kmelia-war/src/main/java/org/silverpeas/components/kmelia/export/DefaultExportFileNameProducer.java

```diff
@@ -24,6 +24,7 @@
 
 import org.silverpeas.core.admin.space.SpaceInst;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.silverpeas.core.admin.space.SpaceInst'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)